public/docs/_examples/cb-third-party-lib/hero-profile/publish.js

```diff
@@ -0,0 +1,59 @@
+// #docregion
+function publish() {
+  var inlineResources = require('./inline-resources');
+
+  // AoT compile
+  var spawnNgc = require( 'child_process' ).spawnSync;
+  var ngc = spawnNgc('./public/docs/_examples/node_modules/.bin/ngc', ['-p', './public/docs/_examples/cb-third-party-lib/hero-profile/tsconfig-aot.json']);
```

Comment:
Although this is the correct path for our tooling, it shouldn't be the path shown to users.